swim/ping_test.go

```diff
@@ -53,14 +54,22 @@ func (s *PingTestSuite) TestPing() {
 }
 
 func (s *PingTestSuite) TestPingFails() {
-	res, err := sendPing(s.node, "127.0.0.1:3003", time.Second)
-	s.Error(err, "expected ping to pail")
+	// Create a channel with no handlers registered. Any requests to this
+	// channel should result in an error being returned immediately.
+	ch, err := tchannel.NewChannel("test", nil)
+	ch.ListenAndServe("127.0.0.1:0")
+	s.Require().NoError(err, "channel must create successfully")
+
+	res, err := sendPing(s.node, ch.PeerInfo().HostPort, time.Second)
+	s.Error(err, "expected ping to fail")
```

Comment:
While we're here, I'd suggest closing the channel when done; avoiding leaking resources may save us some pain in the future.